libs/components/icon/src/lib/modules/icon/icon-svg.component.ts

```diff
@@ -23,7 +23,7 @@ const SIZES = new Map([
  */
 @Component({
   selector: 'sky-icon-svg',
-  imports: [NgClass, NgStyle],
+  imports: [NgClass],
```

Comment:
Angular was yelling that `NgStyle` was extra in Angular 19. I just went ahead and removed it.